From 97af796dd588d4cce7ef5b927fd3d956f8baa16c Mon Sep 17 00:00:00 2001
From: Bastien Nocera <hadess@hadess.net>
Date: Tue, 12 Nov 2013 07:45:43 +0000
Subject: background: Fix compilation with libsocialweb support

The flickr source files were added to the wrong internal shared
library, so that the panel could be linked, but not the test
application.

https://bugzilla.gnome.org/show_bug.cgi?id=710829
---
diff --git a/panels/background/Makefile.am b/panels/background/Makefile.am
index 2766207..58e39aa 100644
--- panels/background/Makefile.am
+++ panels/background/Makefile.am
@@ -49,9 +49,9 @@ libbackground_la_SOURCES =		\
 libbackground_la_LIBADD = $(PANEL_LIBS) $(BACKGROUND_PANEL_LIBS) libbackground-chooser.la
 
 if WITH_LIBSOCIALWEB
-libbackground_la_SOURCES += bg-flickr-source.c bg-flickr-source.h
+libbackground_chooser_la_SOURCES += bg-flickr-source.c bg-flickr-source.h
 INCLUDES += $(SOCIALWEB_CFLAGS)
-libbackground_la_LIBADD += $(SOCIALWEB_LIBS)
+libbackground_chooser_la_LIBADD += $(SOCIALWEB_LIBS)
 endif
 
 noinst_PROGRAMS = test-chooser-dialog
--
cgit v0.9.2
